doc.go

```diff
@@ -350,9 +352,10 @@ Example, hello.go:
     }
 
 A file containing the routes, routes.eskip:
-
-    Random(.05) -> hello("fish?") -> "https://fish.example.org";
-    * -> hello("world") -> "https://www.example.org"
+    
+    hello:
+        Random(.05) -> hello("fish?") -> "https://fish.example.org";
+        * -> hello("world") -> "https://www.example.org"
```

Comment:
actually, both routes should have a name. E.g:

```
random:
        Random(.05) -> hello("fish?") -> "https://fish.example.org";
hello:
        * -> hello("world") -> "https://www.example.org"
```

Reply:
absolutely. I missed it during my copy/paste. 
It's updated now.